Doc/library/dataclasses.rst

```diff
@@ -117,43 +117,43 @@ Module contents
      identical type.  If ``order`` is true and ``eq`` is false, a
      :exc:`ValueError` is raised.
 
-     If the class already defines any of :meth:`~object.__lt__`,
-     :meth:`~object.__le__`, :meth:`~object.__gt__`, or :meth:`~object.__ge__`, then
+     If the class already defines any of :meth:`~!object.__lt__`,
+     :meth:`~!object.__le__`, :meth:`~!object.__gt__`, or :meth:`~!object.__ge__`, then
      :exc:`TypeError` is raised.
 
    - ``unsafe_hash``: If ``False`` (the default), a :meth:`~object.__hash__` method
      is generated according to how ``eq`` and ``frozen`` are set.
 
-     :meth:`~object.__hash__` is used by built-in :meth:`hash()`, and when objects are
+     :meth:`~!object.__hash__` is used by built-in :meth:`hash()`, and when objects are
      added to hashed collections such as dictionaries and sets.  Having a
-     :meth:`~object.__hash__` implies that instances of the class are immutable.
+     :meth:`~o!bject.__hash__` implies that instances of the class are immutable.
      Mutability is a complicated property that depends on the programmer's
-     intent, the existence and behavior of :meth:`~object.__eq__`, and the values of
-     the ``eq`` and ``frozen`` flags in the :func:`dataclass` decorator.
+     intent, the existence and behavior of :meth:`~!object.__eq__`, and the values of
+     the ``eq`` and ``frozen`` flags in the :func:`!@dataclass` decorator.
 
-     By default, :func:`dataclass` will not implicitly add a :meth:`~object.__hash__`
+     By default, :func:`!@dataclass` will not implicitly add a :meth:`~object.__hash__`
      method unless it is safe to do so.  Neither will it add or change an
-     existing explicitly defined :meth:`~object.__hash__` method.  Setting the class
+     existing explicitly defined :meth:`~o!bject.__hash__` method.  Setting the class
      attribute ``__hash__ = None`` has a specific meaning to Python, as
-     described in the :meth:`~object.__hash__` documentation.
+     described in the :meth:`~o!bject.__hash__` documentation.
 
-     If :meth:`~object.__hash__` is not explicitly defined, or if it is set to ``None``,
-     then :func:`dataclass` *may* add an implicit :meth:`~object.__hash__` method.
-     Although not recommended, you can force :func:`dataclass` to create a
-     :meth:`~object.__hash__` method with ``unsafe_hash=True``. This might be the case
+     If :meth:`~o!bject.__hash__` is not explicitly defined, or if it is set to ``None``,
+     then :func:`!@dataclass` *may* add an implicit :meth:`~!object.__hash__` method.
+     Although not recommended, you can force :func:`!@dataclass` to create a
+     :meth:`~!object.__hash__` method with ``unsafe_hash=True``. This might be the case
```

Comment:
```suggestion
     If :meth:`!__hash__` is not explicitly defined, or if it is set to ``None``,
     then :func:`!@dataclass` *may* add an implicit :meth:`!__hash__` method.
     Although not recommended, you can force :func:`!@dataclass` to create a
     :meth:`!__hash__` method with ``unsafe_hash=True``. This might be the case
```